src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -379,6 +379,14 @@ public sealed partial class JsonPropertyNameAttribute : System.Text.Json.Seriali
     }
     public static partial class JsonSerializer
     {
+        public static object ReadValue(ref Utf8JsonReader reader, Type returnType, JsonSerializerOptions options = null) { throw null; }
+        public static TValue ReadValue<TValue>(ref Utf8JsonReader reader, JsonSerializerOptions options = null) { throw null; }
+        //public static void WriteAsValue(object value, Type type, Utf8JsonWriter writer, JsonSerializerOptions options = null) { throw null; }
+        //public static void WriteAsValue<TValue>(TValue value, Utf8JsonWriter writer, JsonSerializerOptions options = null) { throw null; }
+        //public static void WriteAsProperty(ReadOnlySpan<byte> utf8PropertyName, object value, Type type, Utf8JsonWriter writer, JsonSerializerOptions options = null) { throw null; }
+        //public static void WriteAsProperty<TValue>(ReadOnlySpan<char> propertyName, TValue value, Utf8JsonWriter writer, JsonSerializerOptions options = null) { throw null; }
+        //public static void WriteAsProperty<TValue>(string propertyName, TValue value, Utf8JsonWriter writer, JsonSerializerOptions options = null) { throw null; }
+        //public static void WriteAsProperty<TValue>(JsonEncodedText propertyName, TValue value, Utf8JsonWriter writer, JsonSerializerOptions options = null) { throw null; }
```

Comment:
I assume this is WIP and it won't actually be merged like this?

Reply:
Yes, will either remove, or add the APIs. I think I will separate the PRs out.